### DISCUSSION TITLE
“Seeking plots took too long” sources

### DISCUSSION THREAD

**Discussion by ebeng** [martomi/chiadog#345] [Ideas]
Is it possible to include the mount points where these plots are in the notifications?

**Comment by BeepBo0** [comment]
+1 to this! One of the most important things missing from this project.

**Comment by GolDenis72** [comment]
+ can you, please, to make setup for 5/8 sec for search delay in report.
I mean that one:
"Search 🔍:
- average: .......searches
- **over 5s:** .......
- over 15s: ........"
as I understand, new version of chia farmer have a 8sec window for searching plot
